src/main/resources/fester_messages.xml

```diff
@@ -163,4 +163,5 @@
   <entry key="MFS-147">Festerize is outdated, please upgrade to v{} (see README for instructions)</entry>
   <entry key="MFS-148">S3 endpoints healthcheck failed: {})</entry>
   <entry key="MFS-149">S3 endpoint response issue: {})</entry>
+  <entry key="MFS-150">Test CheckEndpointsHandler with request path: {})</entry>
 </properties>
```

Comment:
these last few messages have unmatched closing parens

Reply:
Fixed the typos